Sources/Fuzzilli/FuzzIL/Operations.swift

```diff
@@ -622,6 +622,31 @@ class Reassign: Operation {
     }
 }
 
+/// Destructs an array into n output variables
+class DestructArray: Operation {
+    let indices: [Int]
```

Comment:
Do we want the OperationMutator to be able to modify these (i.e. turn one index into another)? I think that could make sense, although it'll need to ensure the indices are still sorted and contain no duplicates